### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Lul-Abdifan**
###   List Structure and Add Interactive
-  Setup web pack
-  Add  liters configuration
-  Add src directory
   - Add index.js file
   - Add an index.html file
   - Add style.css file
 -Make the page more interactive 
-  Add ReadMe file

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  border-left: 50px solid rgba(179, 172, 172, 0.137);
+  border-top: 10px solid rgba(179, 172, 172, 0.137);
+  border-bottom: 10px solid rgba(179, 172, 172, 0.137);
+  border-radius: 5px;
+}
+
+/* .far {
+  padding-top: 15px;
+  background-color: white;
+} */
```
Comment (AmaduKamara) [inline src/style.css:30]: - [ ] Let's try not to have uncommented code in our codebase as this makes our code dirty and unprofessional. It's a good convention to not have commented unused code in our codebase. Please remove them 👍🏽

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+    print: './src/print.js',
+  },
```
Comment (AmaduKamara) [inline webpack.config.js:9]: - [ ] Webpack is trying to load this `./src/print.js` which is not existing inside the src directory and it's giving an error on page load. Kindly remove this line from your config so the app runs fine with no errors. 👍🏽

Screenshot 2023-03-27 at 5 14 49 PM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @Lul-Abdifan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used array of objects to store todo items 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Lul-Abdifan** [comment]
@AmaduKamara  Thank you for your recent feedback on my work.  Your constructive suggestions were also extremely helpful in identifying areas where I can improve. :smile: happy coding :man_technologist: :smile:

File: webpack.config.js (hunk 2 above)
Comment (Lul-Abdifan) [reply webpack.config.js:9]: I did thank you bro

**Comment by Lul-Abdifan** [comment]
### Required Changes Done
 -  Delete commented codes
 -  Delete unintentionally forgotten code
 @AmaduKamara, Thank you for helping me see my speed.

**Review by vickymarz — APPROVED** [review]
Hi @Lul-Abdifan, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**